### DISCUSSION TITLE
Skyline has panicked Error Code: 2--0069

### DISCUSSION THREAD

**Discussion by Metroflowchart** [Raytwo/ARCropolis#423] [Issues]
System Firmware Version
14.1.2

Atmosphere Version
1.3.2

Smash Version
13.0.1

ARCropolis Version
3.6.2

Can the crash be consistently reproduced?
Yes

At what moment does the crash happen?
Starting up the game

Details:
Problem first occurred when I installed a skin mod, the mod worked fine but when I went to the in game mod manager from the eshop I was met with the error code, I closed the game and then I deleted the mod I installed to see if that would fix the problem, but to no success. Happens every time I boot up ultimate now.

![IMG_3204](https://user-images.githubusercontent.com/113459633/189957733-1d51d535-ba59-47c8-ab4d-8f00ce948caa.png)

**Comment by Raytwo** [comment]
Delete ``sd:/ultimate/arcropolis/config`` and try again.

**Reply by bryzcock-pixel** [reply]
Did not work for me
